src/components/learner-credit-management/SubBudgetCardUtilization.jsx

```diff
@@ -0,0 +1,108 @@
+import PropTypes from 'prop-types';
+import { Col, Skeleton, Card } from '@openedx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import classNames from 'classnames';
+import { formatPrice } from './data';
+import { BUDGET_STATUSES } from '../EnterpriseApp/data/constants';
+
+const SubBudgetCardUtilization = ({
+  isAssignable,
+  isFetchingBudgets,
+  status,
+  available,
+  pending,
+  spent,
+}) => {
+  if (!(
+    status === BUDGET_STATUSES.active
+    || status === BUDGET_STATUSES.expiring
+    || status === BUDGET_STATUSES.retired
+    || status === BUDGET_STATUSES.expired
+  )) {
+    return null;
+  }
```

Comment:
Given the parent component `SubBudgetCard` has the same conditional check against these statuses via `hasBudgetAggregatesSection`, should `SubBudgetCardUtilization` only be rendered when `hasBudgetAggregatesSection` is truthy?

```jsx
// SubBudgetCard

{hasBudgetAggregatesSection() && (
  <SubBudgetCardUtilization ... />
)}
```